README.md

```diff
@@ -79,6 +79,9 @@ optional arguments:
                         adjust image contrast (default: 1.0)
   -l, --loop            loop a single video; otherwise play through the files
                         in the videos directory
+  -o {DEBUG,INFO,WARNING,ERROR,CRITICAL}, --loglevel {DEBUG,INFO,WARNING,ERROR,CRITICAL}
+                         minimum importance-level of messages saved to the
+                         logfile (default: INFO)
 
 Args that start with '--' (eg. -f) can also be set in a config file
 (slowmovie.conf). Config file syntax allows: key=value, flag=true,
```

Comment:
```suggestion
Arguments can also be set in a config file (slowmovie.conf). Config
file syntax allows: key=value, flag=true,
```
I tried using short arguments in the config file, and it works. (I would've aligned the rest of this paragraph, but GitHub doesn't allow commenting on collapsed lines.)